modular_skyrat/modules/customization/modules/surgery/organs/silicon_alt_brains.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/mmi/posibrain/circuit
+	name = "compact AI circuit"
+	desc = "A compact circuit, perfectly dimensioned to fit in the same slot as a cyborgs positronic brain."
+	icon = 'modular_skyrat/master_files/icons/obj/alt_silicon_brains.dmi'
+	icon_state = "circuit"
+	base_icon_state = "circuit"
+	inhand_icon_state = "electronic"
+	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
+
+	// It pains me to copy-paste so much, but I can't do it any other way
+	begin_activation_message = "<span class='notice'>You carefully locate the manual activation switch and start the compact AI circuit's boot process.</span>"
+	success_message = "<span class='notice'>The compact AI circuit pings, and its lights start flashing. Success!</span>"
+	fail_message = "<span class='notice'>The compact AI circuit buzzes quietly, and the golden lights fade away. Perhaps you could try again?</span>"
+	new_mob_message = "<span class='notice'>The compact AI circuit chimes quietly.</span>"
+	recharge_message = "<span class='warning'>The compact AI circuit isn't ready to activate again yet! Give it some time to recharge.</span>"
```

Comment:
IIRC, considering this is a subtype and would inherit these...
you COULD go upstream and change these messages to use [src] in place of the posi's item name 🤔 Someone correct that if I'm wrong tho

Reply:
You can't do that on var init. I could do it via Initialize, but some places use initial for descriptions IIRC.